book/index.md

```diff
@@ -17,7 +17,7 @@ misbehave. Read them at your own risk.
 ::: {.signup}
 :::
 
-Follow this book's [blog](blog/) or
```

Comment:
We should also take down the blog/ URL and add a redirect from it to substack. Agree?

Reply:
Absolutely, good find.